pagopa/pn-frontend-e2e
# PN-9414-e-2-e-destinatario-delegato-rifiuto-delega

### Issue Description
Some PF's tests are failing due to some bad background API integration. In this case the **PN-9414** is failing because it can't find any delegation before revoking one as expected in the test.

### Changes Made:
- add to the `pom.xml` the follow dependencies: `ModelMapper`, `Lombok`, `Gson`
- add `@After` into `Hooks` so each `@DeleghePF` revokes any delagates at the end of the test in order to clear the data preparation for the next re-runs
- add new models/rest classes related to `delegates`

### How to Test
Simply run `mvn clean test -Dcucumber.filter.tags=@TA_PFrifiutoDelega` and verify if the test works fine.
Also a run into AWS CodeBuild is necessary in order to test everything.
